free-2.md

```diff
@@ -1,44 +1,51 @@
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it is deliberate and strict. Unlike English, this DSL will allow little room for ambiguity as it will have fields such as who/what this law is intended for, when this law will be enacted, who/what this law will effect. This DSL addresses a longstanding need for clarity and conciseness in law making.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am currently taking a public policy course off campus and one thing made extremely obvious is how convoluted the legislation process has grown to be. You don't have to be taking the course to understand just how slow the process can be. Although it can be argued ambiguity is a virtue (which is something we discussed in class e.g. the Constition is relevant today only because of its ambiguity), I think there is significant room for optimization.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Law making without B. S.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A legislator may interact with the lanuage by entering statements line by line, with each statement being some rule that adheres to the syntax of the language such that it can be made some logical predicate. Programming would ideally look closer to natural language with a few syntactical oddities (maybe logic symbols) to ensure clarity and enforce rules/logic. I would argue this is the right way to interact with laws because legislators can focus more on the content and less about the semantics.  
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+Ideally, when a program runs, each statement will be processed into a logic statement which could then be compared to the existing set of laws and show similar or related laws. There can also be a field to specify some type of categorization of the law e.g. federal law, state law, local law, etc. Errors that may occur include improperly formatted statements, or conflcits with existing laws, which would be outputted and made apparent to the user.
 
```

Comment:
So would this primarily be used to create new laws or search through existing laws? It seems like you might be able to make something different where the primary purpose is to search current laws to double check court decisions or fact-check presidential debates. I know this is different from what you're describing, but it might be something to consider.